client/src/components/App.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import TopBar from './TopBar.jsx'
+import VideoPLayer from './VideoPLayer.jsx'
+import StreamDetail from './StreamDetail.jsx'
+import '../styles.css';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      streamerId: 3,
+      streamerData: {}
+    }
+    this.commasWithNumber = this.commasWithNumber.bind(this)
+  }
+
+  componentDidMount() {
+    const streamerId = this.props.streamerId || this.state.streamerId
+    fetch('/api/livestream/' + streamerId)
+    .then(res => res.json())
+    .then(data => {
+      data[0].total_views = this.commasWithNumber(data[0].total_views)
+      data[0].watching_now = this.commasWithNumber(data[0].watching_now)
+      console.log(typeof data[0].video_url)
+      this.setState({
+        streamerId: data[0].id,
+        streamerData: data[0]
+      })
+    })
+  }
+
+  commasWithNumber(num) {
```

Comment:
This is interesting. I need to learn regular expressions. :)

Reply:
stack overflow haha. I need to learn it too.